specification/netapp/resource-manager/Microsoft.NetApp/stable/2019-07-01/netapp.json

```diff
@@ -0,0 +1,2355 @@
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.NetApp/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "summary": "Describes the Resource Provider",
+        "description": "Lists all of the available Microsoft.NetApp Rest API operations",
+        "operationId": "Operations_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "x-ms-pageable": {
+          "nextLinkName": null
+        },
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/OperationListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed."
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.NetApp/locations/{location}/checkNameAvailability": {
+      "post": {
+        "summary": "Check resource name availability",
+        "description": "Check if a resource name is available.",
+        "operationId": "CheckNameAvailability",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "description": "Name availability request.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ResourceNameAvailabilityRequest"
+            },
+            "x-ms-client-flatten": true
+          },
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/location"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ResourceNameAvailability"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed."
+          }
+        },
+        "x-ms-examples": {
+          "CheckNameAvailability": {
+            "$ref": "examples/CheckNameAvailability.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.NetApp/locations/{location}/checkFilePathAvailability": {
+      "post": {
+        "summary": "Check file path availability",
+        "description": "Check if a file path is available.",
+        "operationId": "CheckFilePathAvailability",
```

Comment:
Add Noun_ prefix here too, it should match with `CheckNameAvailability`

Reply:
ok. Note that this does not relate to the change made and exists in earlier versions. Can it therefore be delayed until the next version since this swagger is already now overdue?

Reply:
I see.  If there are previous releases with this pattern then let's just leave it as-is in order to not introduce any breaking changes.